test/platform-stats.test.js

```diff
@@ -50,36 +52,44 @@ describe('platform-stats', () => {
 
   describe('updateDailyStationStats', () => {
     it('updates daily station stats for today with multiple measurements', async () => {
-      const validStationId2 = VALID_STATION_ID.slice(0, -1) + '1'
       const honestMeasurements = [
         { ...VALID_MEASUREMENT, stationId: VALID_STATION_ID },
-        { ...VALID_MEASUREMENT, stationId: validStationId2 }
+        { ...VALID_MEASUREMENT, stationId: VALID_STATION_ID_2 }
       ]
 
       await updateDailyStationStats(pgClient, honestMeasurements)
 
       const { rows } = await pgClient.query(`
-        SELECT station_id, day::TEXT FROM daily_stations
+        SELECT station_id, day::TEXT, accepted_measurement_count
+        FROM daily_stations
         ORDER BY station_id`
       )
       assert.strictEqual(rows.length, 2)
       assert.deepStrictEqual(rows, [
-        { station_id: VALID_STATION_ID, day: today },
-        { station_id: validStationId2, day: today }
+        { station_id: VALID_STATION_ID, day: today, accepted_measurement_count: 1 },
+        { station_id: VALID_STATION_ID_2, day: today, accepted_measurement_count: 1 }
       ])
     })
 
-    it('ignores duplicate measurements for the same station on the same day', async () => {
+    it('counts honest measurements for the same station on the same day', async () => {
       const honestMeasurements = [
         { ...VALID_MEASUREMENT, stationId: VALID_STATION_ID },
-        { ...VALID_MEASUREMENT, stationId: VALID_STATION_ID }
+        { ...VALID_MEASUREMENT, stationId: VALID_STATION_ID },
+        { ...VALID_MEASUREMENT, stationId: VALID_STATION_ID_2 }
       ]
 
       await updateDailyStationStats(pgClient, honestMeasurements)
 
-      const { rows } = await pgClient.query('SELECT station_id, day::TEXT FROM daily_stations')
-      assert.strictEqual(rows.length, 1)
-      assert.deepStrictEqual(rows, [{ station_id: VALID_STATION_ID, day: today }])
+      const { rows } = await pgClient.query(`
+        SELECT station_id, day::TEXT, accepted_measurement_count
+        FROM daily_stations
+        ORDER BY station_id`
+      )
+      assert.strictEqual(rows.length, 2)
+      assert.deepStrictEqual(rows, [
+        { station_id: VALID_STATION_ID, day: today, accepted_measurement_count: 2 },
+        { station_id: VALID_STATION_ID_2, day: today, accepted_measurement_count: 1 }
+      ])
     })
 
     it('ignores measurements without .stationId', async () => {
```

Comment:
@PatrickNercessian this test is failing now. Could you please investigate?

```
  1) platform-stats
       updateDailyStationStats
         ignores measurements without .stationId:
     error: null value in column "station_id" of relation "daily_stations" violates not-null constraint
      at /Users/bajtos/src/spark/evaluate/node_modules/pg/lib/client.js:526:17
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async updateDailyStationStats (file:///Users/bajtos/src/spark/evaluate/lib/platform-stats.js:32:3)
      at async Context.<anonymous> (file:///Users/bajtos/src/spark/evaluate/test/platform-stats.test.js:101:7)
```

Reply:
I merged the latest version from `main` into your feature branch and resolved the merge conflict, hopefully correctly. Please remember to `git pull` the updates!

Reply:
Ah apologies, will do.

Addressed in https://github.com/filecoin-station/spark-evaluate/pull/200/commits/1c9884ec6a6e9c12af294cd883f3a1de2c1805a8